README.md

```diff
@@ -1 +1,62 @@
-# -Global-Warming
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Global Warming / Gapstone one
+
+> Global Warming responsive website.
+> This version contain main and about page.
+> The next version will contains more pages.
```

Comment:
Kindly add a screenshot of your website to make your README file more professional.